gradle.properties

```
@@ -36,7 +36,7 @@ stdlibJwtVersion=2.8.0
 stdlibOAuth2Version=2.8.0
 
 # Level 05
-stdlibHttpVersion=2.8.0
+stdlibHttpVersion=2.8.3-20231012-153400-b73fc00
```

Update http version after release
```suggestion
stdlibHttpVersion=2.8.3
```